### PR TITLE
Add StationUI

### PR DESCRIPTION
**Pull request by APB9785**
This installs StationUI into BeaconLiveAdmin and replaces several components such as `.button`, `.input`, and `.icon`

### CODE REVIEW

**Review by cibernox — APPROVED** [review]

File: lib/beacon/live_admin/web.ex [hunk 1]
```diff
@@ -62,6 +62,7 @@ defmodule Beacon.LiveAdmin.Web do
 
   defp html_helpers do
     quote do
+      use Beacon.LiveAdmin.StationUI.HTML
```
Comment (leandrocp) [inline lib/beacon/live_admin/web.ex:65]: We can import directly from the StationUI dep once it gets published

Comment (APB9785) [reply lib/beacon/live_admin/web.ex:65]: you mean, just inject the js/css and call the component modules remotely?

Comment (leandrocp) [reply lib/beacon/live_admin/web.ex:65]: Exactly, unless we want to have fine grained control over updates on those components.

Comment (leandrocp) [reply lib/beacon/live_admin/web.ex:65]: Or depending on how UXD want to manage changes, either use stock components or customize. Either way we have some options here but for now only vendor will work anyway.

**Comment by APB9785** [comment]
putting this back to draft because it looks pretty bad as-is, needs a bit of care before merging

**Comment by APB9785** [comment]
currently broken: going into a page and then the "schema" or "revisions" tabs (see navbar)

File: lib/beacon/live_admin/components/station_ui/html/input.ex [hunk 2]
```diff
@@ -938,7 +938,7 @@ defmodule Beacon.LiveAdmin.StationUI.HTML.Input do
           value={Form.normalize_value(@type, @value)}
           disabled={@disabled}
           class={[
-            IO.inspect(@class, label: "textfield class"),
```
Comment (cibernox) [inline lib/beacon/live_admin/components/station_ui/html/input.ex]: Watch this inspect call!

Comment (APB9785) [reply lib/beacon/live_admin/components/station_ui/html/input.ex]: TY!

**Review by leandrocp — APPROVED** [review]
Tested by deploying beacon_demo with this branch

**Comment by cibernox** [comment]
👏

**Pull request merged**